core/modules/simpletest/tests/module.test

```diff
@@ -332,6 +332,32 @@ class ModuleUninstallTestCase extends BackdropWebTestCase {
     $this->assertNoText('Uninstall Locale module', 'Module Locale has been successfully uninstalled');
   }
 
+
+  /**
+   * Test that uninstalling Comment and then immediately adding a node works.
+   */
+  public function testUninstallCommentAddNode() {
+    $admin_user = $this->backdropCreateUser(array(
+      'administer modules',
+      'bypass node access',
+    ));
+    $this->backdropLogin($admin_user);
+    module_disable(array('comment'), FALSE);
```

Comment:
This works, but should we be doing it via the UI instead (`$this->backdropPost('admin/modules', $edit, t('Save configuration'));`), same as below with the uninstallation of the module? ...if not, then should we be also doing the uninstallation manually via `backdrop_uninstall_modules()`?

Reply:
This test tries to capture problems when uninstalling the module, so disabling isn't actually relevant here. I chose module_disable for efficiency reasons (to not waste test run resources).

The uninstall, however, should be done in a realistic way - via UI. That's why I chose backdropPost here.